.github/workflows/release.yaml

```diff
@@ -267,31 +261,66 @@ jobs:
         run: |
           hack/build/ci/install-cgo-dependencies.sh
       - name: Generate release notes
-        id: generate_notes
+        shell: bash
+        env:
+          PRE_RELEASE: ${{ contains(github.ref, '-rc.') }}
         run: |
           hack/build/ci/generate-release-notes.sh
       - name: Generate manifests
         env:
-          IMAGE: ${{ matrix.url }}/${{ secrets[matrix.repository] }}
-          TAG: ${{ needs.prepare.outputs.version }}
-          CHART_VERSION: ${{ needs.prepare.outputs.version }}
-        run: make manifests
-      - name: Rename manifests
+          version: ${{ needs.prepare.outputs.version }}
+          platforms: |
+            kubernetes
```

Comment:
it would make sense to define platforms in a matrix with each of the platform specific fields (image, generated file, ...) specified there and not spread around in different parts of the script. This could simplify the `generate-release-manifests` script and not require a loop there.

The helm package is only required once and should be done in a separate job, so we'd have the following jobs:
1. create manifests
    - kubernets
    - openshift
    - gke-autopilot 	
2. create helm package
3. create pre-release